app/forms/assessor_interface/create_note_form.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class AssessorInterface::CreateNoteForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveRecord::AttributeAssignment
+
+  attr_accessor :application_form, :author
+  attribute :text, :string
+
+  validates :application_form, :author, :text, presence: true
+
+  def save!
+    return false unless valid?
+
+    ActiveRecord::Base.transaction do
+      note = Note.create!(application_form:, author:, text:)
+
+      TimelineEvent.create!(
+        application_form:,
+        event_type: "note_created",
+        creator: author,
+        note:,
```

Comment:
We could use the eventable association here. If we're going to add note explicitly, then maybe we should change the other eventable things (or thing at the moment I think) to be explicit too and get rid of the polymorphic association. This does seem like a good place to use one to me though but happy to discuss.

Reply:
Fair enough. Let's do that 👍